pkg/arvo/app/publish.hoon

```diff
@@ -1963,7 +2042,11 @@
               date-created.data.del
           ==
       ==
+    =/  book  (~(get by books) host.del book.del)
     =^  cards  state
+      ?~  book
+        (emit-updates-and-state host.del book.del data.del del sty)
+      =.  data.del  (merge-notebooks u.book data.del)
       (emit-updates-and-state host.del book.del data.del del sty)
```

Comment:
You might want `=?  data.del  (~(has by books) host.del book.del)` here instead. (Also, I generally prefer putting the `=/` at a more local "scope", closer to actual use.)